pubspec.yaml

```diff
@@ -5,7 +5,6 @@ version: 2.3.0+020300
 
 environment:
   sdk: ">=2.6.0 <3.0.0"
-  flutter: ^1.16.0
```

Comment:
I think this should still be specified

Reply:
I think it does not make sense as we're bound to master. So it's not actually accurate. If we could set this to be master then it would make sense.